src/components/ButtonsBox.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] As mentioned by the previous reviewer and for best practices, kindly make sure you don't disable linters errors, but them instead. This will make your code base more professional and has clean code :100: